history/units/TRA_3236.txt

```diff
@@ -0,0 +1,339 @@
+	}
+	division = {
+		location = 1214
+		division_template = "Jager Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 614
+		division_template = "Jager Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 506
+		division_template = "Armor Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 2102
+		division_template = "Armor Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 794
+		division_template = "Motorized Division"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 1722
+		division_template = "Motorized Division"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 576
+		division_template = "Light Motorized Division"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 2013
+		division_template = "Light Motorized Division"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 1631
+		division_template = "Light Motorized Division"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 1012
+		division_template = "Light Motorized Division"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 866
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 576
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 637
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 879
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 1863
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 1135
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 1503
+		division_template = "Infantry Brigade with MRLS"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 949
+		division_template = "Infantry Brigade with MRLS"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 794
+		division_template = "Infantry Brigade with MRLS"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 2021
+		division_template = "Infantry Brigade with MRLS"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
```

Comment:
Two extra infantry and two missing artillery. Should these be the "Infantry with MRLS"?

Reply:
Yes, I w/ MLRS = artillery